gossip/sfc_index.go

```diff
@@ -296,6 +298,20 @@ func (s *Service) processSfc(block *inter.Block, receipts types.Receipts, blockF
 				constants.OfflinePenaltyThreshold.Period = inter.Timestamp(period.Uint64())
 				s.app.SetSfcConstants(epoch, constants)
 			}
+			if l.Topics[0] == sfcpos.Topics.UpdatedMinGasPrice && len(l.Data) >= 32 {
+				minGasPrice := new(big.Int).SetBytes(l.Data[0:32])
+				constants := s.app.GetSfcConstants(epoch)
+				constants.MinGasPrice = minGasPrice
+				s.app.SetSfcConstants(epoch, constants)
+			}
+
+			if l.Topics[0] == sfcpos.Topics.NetworkUpgradeActivated && len(l.Data) >= 32 {
+				minVersion := new(big.Int).SetBytes(l.Data[0:32])
+				if minVersion.Sign() > 0 {
```

Comment:
minVersion should be compared with currentVersion

Reply:
Suggest https://github.com/uprendis/go-lachesis/pull/2

Reply:
Yeah, you're right, I didn't include comparison with an actual version because it's the first version with this feature, so we cannot target previous versions anyway. Your suggestion looks good